MSAL/src/instance/MSALDeviceInfoProvider.m

```diff
@@ -41,6 +41,22 @@ @implementation MSALDeviceInfoProvider
 - (void)deviceInfoWithRequestParameters:(MSIDRequestParameters *)requestParameters
                         completionBlock:(MSALDeviceInformationCompletionBlock)completionBlock
 {
+    void (^deviceInfoCompletionHandler)(MSALDeviceInformation *, NSError *) = ^void(MSALDeviceInformation *msalDeviceInfo, NSError *error)
```

Comment:
will it be better to call it `fillDeviceInfoCompletionBlock`? Besides, we use `handler` to mean something else in our code :)